client/i18n/locales/english/translations.json

```diff
@@ -416,6 +416,21 @@
     "help-translate-link": "Help us translate.",
     "project-preview-title": "Here's a preview of what you will build",
     "github-required": "<0>Create a GitHub</0> account if you don't have one. You'll need it when you create the virtual Linux server machine. This process may take a few minutes.",
+    "gitpod": {
+      "intro": "This course runs in a virtual Linux machine using Gitpod. Follow these instructions to start the course:",
+      "step-1": "<0>Create a GitHub</0> account if you don't have one",
+      "step-2": "Click the start button below",
+      "step-3": "Login to Gitpod with your GitHub account if you aren't already",
+      "step-4": "Once the virtual Linux machine is finished loading, start the CodeRoad extension by:",
+      "step-5": "Clicking the \"hamburger\" menu near the top left of the VSCode window,",
+      "step-6": "Going to the \"View\" menu,",
+      "step-7": "Clicking on the \"Command Palette\" option,",
+      "step-8": "and running the \"CodeRoad: Start\" command",
+      "step-9": "Follow the instructions in CodeRoad to complete the course",
+      "continue-project": "Clicking the button below will start a new project. If you have previously started the course, go to <0>your Gitpod dashboard</0> to continue.",
+      "learn-more": "Learn more about Gitpod workspaces and how to <0>maximize your free usage.</0>",
+      "logout-warning": "If you log out of freeCodeCamp before you complete this entire course, your progress will not be saved to your freeCodeCamp account."
```

Comment:
This is confusing for me. If I log out of `/learn`, complete the project on Gitpod, my progress is not recorded?

I thought tokens persisted with the account.

---

Also, I realise we specifically use `curriculum` and `course`, but to a Camper, I suspect they might read "...before your complete this entire curriculum..." - not just the single project.

Maybe we should use the word `project`?

Reply:
> I thought tokens persisted with the account.

They do, they're part of your user record. If the camper actively deletes their token, then the existing workspaces will be unable to submit, but otherwise they should be fine.

Reply:
No, the [tokens are deleted when you log out.](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/api-server/src/server/boot/authentication.js#L71)

Reply:
Sorry, I forgot about that.

We should guide users about how to fix this, because it's going to be very frustrating. Fortunately it seems `gp env CODEROAD_WEBHOOK_TOKEN=<users-token>` should do the trick, but it does mean we need to show them their token.

Reply:
I think that's going to need to be a post MVP thing. I do have code somewhere in an old PR that displays the tokens.

Reply:
Okay, I've tracked it in an issue.

Reply:
Thanks. This will also help for letting users run this locally and still get credit.